home/modules/ROOT/pages/sdk.adoc

```diff
@@ -1,5 +1,5 @@
 = SDKs & Connectors
-:page-aliases: sdks:intro.adoc
+:page-aliases: sdks:intro.adoc,7.0@server:sdk:overview.adoc,7.1@server:sdk:overview.adoc,7.2@server:sdk:overview.adoc,6.6@server:install:install-client-server.adoc,7.0@server:install:install-client-server.adoc,7.1@server:install:install-client-server.adoc,7.2@server:install:install-client-server.adoc
```

Comment:
cross component-version aliases work? 😱 
assume you've checked this locally, so 👍

Reply:
Yes - we use them in SDK.
e.g.
https://github.com/couchbase/docs-sdk-java/blob/release/3.4/modules/project-docs/pages/sdk-release-notes.adoc
https://github.com/couchbase/docs-sdk-java/blob/release/3.4/modules/howtos/pages/concurrent-document-mutations.adoc